zip-0316.rst

```diff
@@ -235,6 +247,15 @@ though it were absent.
   shielded support without re-acquiring counterparty UAs. If they are
   re-acquired, the user flow and usability will be minimally disrupted.
 
+Experimental Usage
+------------------
+
+Unified Addresses and Unified Viewing Keys must be able to include
+Receivers and Viewing Keys of experimental types, possibly alongside
+non-experimental ones. These experimental Receivers or Viewing Keys
+must be used only by wallets whose users have explicitly opted into
+the corresponding experiment.
+
 Viewing Keys
 ------------
 
```

Comment:
> Therefore, a UFVK or UIVK should be able to include a Transparent Address.

str4d: We can't do diversification for transparent keys, ends up duplicating the same t-key across diversified shielded sapling/orchard keys, linking them. We need a 'stream of unlinkable addresses' , (extended pubkey)

daira: have to be separate derivation trees for transparent/sapling/orchard because they all use different curves

You can't have a ViewingKey that includes P2SH.

Reply:
I will resolve this.